chain/network/src/peer_manager.rs

```diff
@@ -347,14 +348,14 @@ impl PeerManagerActor {
         addr: Addr<Peer>,
         ctx: &mut Context<Self>,
     ) {
-        near_performance_metrics::actix::run_later(ctx, WAIT_FOR_SYNC_DELAY, move |act, ctx2| {
+        near_performance_metrics::actix::run_later(ctx, WAIT_FOR_SYNC_DELAY, move |act2, ctx2| {
             if peer_type == PeerType::Inbound {
-                act.routing_table_pool
+                act2.routing_table_pool
                     .send(RoutingTableMessages::AddPeerIfMissing(peer_id, None))
-                    .into_actor(act)
-                    .map(move |response, act2, _ctx| match response {
+                    .into_actor(act2)
+                    .map(move |response, act3, _ctx3| match response {
                         Ok(RoutingTableMessagesResponse::AddPeerResponse { seed }) => {
-                            act2.start_routing_table_syncv2(addr, seed)
+                            act3.start_routing_table_syncv2(addr, seed)
```

Comment:
I'd personally just use `act` and `ctx` everywhere, as shadowing is idiomatic Rust, but no strong opinion.

Reply:
Thanks, that's a good idea.